arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTrace.storyboard

```diff
@@ -85,7 +86,8 @@
                                 </items>
                             </toolbar>
                         </subviews>
-                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <viewLayoutGuide key="safeArea" id="Qfl-lK-vPg"/>
+                        <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
```

Comment:
To keep consistent among all samples, the root view will use `tertiarySystemBackground` so that the toolbar are displayed as a lighter gray color in Dark Mode.